tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "target": "ESNext",
     "useDefineForClassFields": true,
+    "types": ["node", "vite/client"],
```

Comment:
Something I found out is that including this config option disables type acquisition from `@types`. So that's why the typechecking is failing. The line in [src/vite-env.d.ts](https://github.com/matrix-org/thirdroom/pull/30/files#diff-04727777ee54a542b21552456d6e0ee929e52d1b40c5916419c95152caf95947) is doing that work. We should have `@types/node` installed. is that not working for you?